lib/hook-template.js

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env node
 
-try {
```

Comment:
try-catch is possible here if we use sync version of exec.

> Remove node_modules directory. -> you cannot make a commit ;)
what do you mean?)